gnovm/tests/files/zrealm_borrow7.gno

```diff
@@ -0,0 +1,21 @@
+// PKGPATH: gno.land/r/borrow
+package borrow
+
+import "gno.land/p/demo/tests/p_crossrealm"
+
+func init() {
+	p_crossrealm.F.Modify2()
+	println("==================================================================")
+}
+
+func main() {
+	println("F after modify: ", p_crossrealm.F)
+	println("Err after modify: ", p_crossrealm.Err)
+}
+
+// Output:
+// &(struct{(1 int)} gno.land/p/demo/tests/p_crossrealm.Foo)
+// Err 2
+// ==================================================================
+// F after modify:  readonly(&(struct{(0 int)} gno.land/p/demo/tests/p_crossrealm.Foo))
+// Err after modify:  Err 1
```

Comment:
it's possible crossing into p/ by soft crossing and mutate, then the modification will be discarded.

but it's not possible by top level functions, odd?